MekHQ/src/mekhq/campaign/finances/Money.java

```diff
@@ -102,6 +102,10 @@ public Money absolute() {
     }
 
     public Money plus(Money amount) {
+        if (amount == null) {
+            return plus(0L);
```

Comment:
This ensures we get a clone of the underlying `BigMoney` instance and don't share it between two `Money` instances.